doc/classes/@GlobalScope.xml

```diff
@@ -2726,6 +2726,9 @@
 			hint_string = "%s:%s:" % [TYPE_ARRAY, TYPE_REAL] # Two-dimensional array of floats.
 			hint_string = "%s/%s:Resource" % [TYPE_OBJECT, TYPE_OBJECT] # Array of resources.
 			hint_string = "%s:%s/%s:Resource" % [TYPE_ARRAY, TYPE_OBJECT, TYPE_OBJECT] # Two-dimensional array of resources.
+			hint_string = "%s/%s:Zero,One,Two" % [TYPE_INT, TYPE_INT] # Array of integers (see PROPERTY_HINT_ENUM).
+			hint_string = "%s/%s:Zero,One,Three:3,Four,Six:6" % [TYPE_INT, TYPE_INT] # Array of integers (see PROPERTY_HINT_ENUM).
+			hint_string = "%s/%s:Zero,One,Two" % [TYPE_STRING, TYPE_INT] # Array of strings (see PROPERTY_HINT_ENUM).
```

Comment:
The syntax still confuses the heck out of me, why do we specify the type twice and what does it differ for `TYPE_STRING, TYPE_INT` but not in other cases?

The initial description for this constant likely needs more work, it refers to `Resource` specifically when this doesn't seem to be a limitation? And it talks about `:` and `/` but not what they mean.

Reply:
It already works this way. :upside_down_face:
#60516 has "Release blocker" status for some reason.

Reply:
That was wrong, it shouldn't have.

IMO this should be left for 4.1, it's not critical at this stage. #60516 was labelled as a regression because the feature existed in 3.x, but it's not a deal breaker if it's not in 4.0.

Reply:
It is not type twice, it is the type and then the **hint**. So it should be `TYPE_INT` and then `PROPERTY_HINT_ENUM` (which is currently is `2`, the same as `TYPE_INT`)